components/profile/index.jsx

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import React, { useState } from "react";
+import T from "prop-types";
+import { vcard } from "rdf-namespaces";
+import Link from "next/link";
+import {
+  Avatar,
+  Box,
+  Paper,
+  InputLabel,
+  createStyles,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import clsx from "clsx";
+import { useBem } from "@solid/lit-prism-patterns";
+import { Container } from "@inrupt/prism-react-components";
+
+import { CombinedDataProvider, Text, Image } from "@inrupt/solid-ui-react";
+
+import ContactInfoTable from "./contactInfoTable";
+import styles from "./styles";
+
+const useStyles = makeStyles((theme) => createStyles(styles(theme)));
+
+export default function Profile(props) {
+  const { profileIri, editing } = props;
+  const [error, setError] = useState(null);
+
+  const classes = useStyles();
+  const bem = useBem(classes);
+
+  // TODO replace with toast error or something?
+  if (error) {
+    return error.toString();
+  }
+
+  return (
+    <Container>
+      <Box mt={2} mb={2}>
```

Comment:
Didn't the `BackToNav` component in Prism work as intended?

Reply:
The colors for the link is not correct - I think the use of Link doesn't apply the CSS class properly.

To see how you can use the BackToNav component, [check out this part](https://github.com/inrupt/pod-browser/commit/aa22d67c166daf456f32d0170151964da0a99157#diff-aa354bb30ce433a020c45f3ad127dae9cf1e92c78b78e6a6a4a7d2e5594312ddR33-R41).